presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftMetastoreClient.java

```diff
@@ -71,6 +72,9 @@ void dropTable(String databaseName, String name, boolean deleteData)
     void alterTable(String databaseName, String tableName, Table newTable)
             throws TException;
 
+    void alterTableWithEnvironmentContext(String databaseName, String tableName, Table newTable, EnvironmentContext context)
```

Comment:
remove `void alterTable(String databaseName, String tableName, Table newTable)`, it's unused now